### PR TITLE
VTK ExprTk cmake patch

### PR DESCRIPTION
**Pull request by ArashPartow**
This patch is intended to unblock the following PR: https://github.com/microsoft/vcpkg/pull/29665

and is based on this VTK MR: https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9981

### CODE REVIEW

**Comment by ArashPartow** [comment]
Question: Should the

File: ports/vtk/FindExprTk.patch [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Author: partow@gmail.com
+Date:   Sat Feb 22 00:00:00 2023 +0000
+
+    Make FindExprTk more robust
+
+    The previous version of this module failed due to trivial changes
+    in the way the "version" variable was declared in exprtk.hpp.
+    Add a fallback so that, if exprtk is found, the reported version
+    is never less than 2.7.
+
```
Comment (Neumann-A) [inline ports/vtk/FindExprTk.patch:11]:
```suggestion
```

Comment (Neumann-A) [reply ports/vtk/FindExprTk.patch:11]: Alternatively download https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9981.diff in the portfile.

Comment (dg0yt) [reply ports/vtk/FindExprTk.patch]: Is this a stable download or generated on the fly?

**Review by FrankXie05 — APPROVED** [review]

**Comment by FrankXie05** [comment]
Need modify the hash. 

> _This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!_
> 
> PRs must add only one version and must not modify any published versions
> When making any changes to a library, the version or port-version in `vcpkg.json` or `CONTROL` must be modified.
> 
> ```
> error: checked-in files for vtk have changed but the version was not updated
> version: 9.2.0-pv5.11.0#3
> old SHA: 35bdd05c8d99bc4933c899b76357830bfeebe01a
> new SHA: 4417f03f05c44af8c50a90a7e7e00fb43efe5979
> Did you remember to update the version or port version?
> Use --overwrite-version to bypass this check
> ***No files were updated***
> ```

**Comment by FrankXie05** [comment]
@ArashPartow This PR  https://github.com/microsoft/vcpkg/pull/29656 updates the `port-version`, you need to increment it once:
`"port-version": 5,`

**Pull request closed without merging**